Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -87,6 +87,10 @@ def genGridOnPoints(data1,gm,deep=True,grid=None,geo=None):
     if grid is None:
       x = g.getLongitude()[:]
       y = g.getLatitude()[:]
+      xm = x[0]
+      xM = x[-1]
+      ym = y[0]
```

Comment:
@aashish24 I knpw, I got your point the first time. Was just teasing you :wink: I will try to fix the variable names from now on. I promise I wil "try".